flutter_web_auth_2/android/src/main/AndroidManifest.xml

```diff
@@ -1,3 +1,9 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-  package="com.linusu.flutter_web_auth_2">
+    package="com.linusu.flutter_web_auth_2">
+
+    <queries>
+        <intent>
+            <action android:name="android.support.customtabs.action.CustomTabsService" />
```

Comment:
Is this enough? Doesn't the user need to specify this on his own?
I have no idea - so please just resolve this comment as done if it really is enough as-is :)

Reply:
yes , added this  then user doesn't need add this on there app AndroidManifest.xml.